eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -533,9 +533,12 @@ class Router(val nodeParams: NodeParams, watcher: ActorRef, initialized: Option[
       sender ! d
       stay
 
-    case Event(FinalizeRoute(partialHops), d) =>
+    case Event(FinalizeRoute(partialHops, assistedRoutes), d) =>
+      val assistedChannels: Map[ShortChannelId, AssistedChannel] = assistedRoutes.flatMap(toAssistedChannels(_, partialHops.last, 0 msat)).toMap
```

Comment:
This is more a remark than a request for change: using `0 msat` here causes the fake channel update to have `htlc_maximum_msat` equal to `RoutingHeuristics.CAPACITY_CHANNEL_LOW`. This is not a problem but it can create routes that are technically unusable because the last hop max htlc is smaller than the amount we want to send, but when using send to route we don't really check that anymore. Maybe a small explanatory comment here would be good.

Reply:
True that it's a bit subtle, I'll add a comment :)